datasets/us/co/med_exclusions/crawler.py

```diff
@@ -0,0 +1,99 @@
+from typing import Dict
+from rigour.mime.types import XLSX
+from openpyxl import load_workbook
+from datetime import datetime
+import re
+
+from zavod import Context, helpers as h
+
+
+def crawl_item(row: Dict[str, str], context: Context):
+
+    name = row.pop("provider_name")
+
+    if not name:
+        return
+
+    entity = context.make("LegalEntity")
+    entity.id = context.make_id(name, row.get("npi"))
+    entity.add("name", name)
+
+    if row.get("npi") != "N/A":
+        npis = re.split(r"; |& ", row.pop("npi"))
+
+        for npi in npis:
+            entity.add("npiCode", npi)
+    else:
+        row.pop("npi")
+
+    sanction = h.make_sanction(context, entity)
+    termination_effective_date = row.pop("termination_effective_date")
+    sanction.add("startDate", termination_effective_date)
+    sanction.add("reason", row.pop("termination_authority"))
+
+    if "reinstatement_effective_date" in row:
+        reinstatement_effective_date = row.pop("reinstatement_effective_date")
+    else:
+        reinstatement_effective_date = None
+
+    if reinstatement_effective_date:
+        target = (
+            datetime.strptime(reinstatement_effective_date, "%Y-%m-%d")
+            >= datetime.today()
+        )
+        sanction.add("endDate", reinstatement_effective_date)
+    else:
+        target = True
+
+    if target:
+        entity.add("topics", "debarment")
+
+    dba = row.pop("doing_business_as_name")
+    if dba:
+        company = context.make("Company")
+        company.id = context.make_id(dba)
+
+        if target:
+            company.add("topics", "debarment")
+
+        # link = context.make("UnknownLink")
+        # link.id = context.make_id(dba)
```

Comment:
the link ID and company ID would end up the same unless you add something to the link

```suggestion
        link.id = context.make_id(entity.id, company.id)
```